ci/tasks/light-aws/run-upload-test.yml

```
@@ -0,0 +1,17 @@
+  - name: bosh-cpi-src
+  - name: bosh-stemcells-ci
+  - name: environment
+  - name: stemcell
+  - name: director-state
+  - name: builder-src
+
+run:
+  path: bosh-stemcells-ci/ci/tasks/light-aws/run-upload-test.sh
+
+params:
+  BOSH_DEBUG_LEVEL:     info
```

YAMLlint (1.38.0)

[error] too many spaces after colon

(colons)